comma/__init__.py

```diff
@@ -0,0 +1,64 @@
+import os
+import json
+import pandas as pd
+
+
+PARAMS_INDIVIDUAL = 'params_individual.json'
+PARAMS_MODEL = 'params_model.json'
+PARAMS_IPF_WEIGHTS = "ipf_weights.csv"
+
+
+def read_json_as_dict(fpath: str):
+    """Read content of a JSON file as Python dictionary.
+
+    Args:
+        fpath (str): file path.
+
+    Returns:
+        dict: result dictionary.
+    """
+    assert os.path.isfile(fpath), 'File not found: %s.' % fpath
+    with open(fpath) as file:
+        data = json.load(file)
+    return data
```

Comment:
This function is redundant, it just calls `json.load()`. Whenever this function is called, `json.load()` should be called directly.